core/src/navigation/ProductSwitcher.html

```diff
@@ -7,8 +7,26 @@
   <div class="fd-product-switch">
     <div class="fd-popover fd-popover--right">
       <div class="fd-popover__control" on:click|stopPropagation="{() => {}}">
-        <!-- default: sap-icon--grid -->
-
+        {#if LuigiConfig.getConfigBooleanValue('navigation.addNavHrefs')}
+        <a
+          href="/"
```

Comment:
Just wondering if this is necessary or redundant, I would not usually right click on this one to get home. But maybe that's some kind of standard ❔

Reply:
On a side note, should we think about having this enabled on mobile version as well, or is that not relevant 🤔